corehq/apps/userreports/expressions/json_query_specs.py

```diff
@@ -0,0 +1,76 @@
+import itertools
+from dimagi.ext.jsonobject import JsonObject, DictProperty, StringProperty
+from corehq.apps.userreports.exceptions import BadSpecError
+from corehq.apps.userreports.specs import TypeProperty
+from .utils import SUPPORTED_UCR_AGGREGATIONS, aggregate_items
+
+
+class FilterItemsExpressionSpec(JsonObject):
+    type = TypeProperty('filter_items')
+    items_expression = DictProperty(required=True)
```

Comment:
hmm. actually i don't know if we support list literals. you could try it. "abc" would be supported, if you ever wanted to iterate over letters...